src/state/slices/portfolioSlice/selectors.ts

```diff
@@ -138,7 +138,7 @@ export const selectPortfolioLoadingStatus = createSelector(
   selectPortfolioLoadingStatusGranular,
   (portfolioLoadingStatusGranular): PortfolioLoadingStatus => {
     const vals = values(portfolioLoadingStatusGranular)
-    if (vals.every(val => val === 'loading')) return 'loading'
+    if (vals.some(val => val === 'loading')) return 'loading'
```

Comment:
treat as loading if any accounts are still "loading"

Reply:
Note, since we won't go to l.142 anymore until all are finished loading, this means we won't have degraded states anymore until *all* are finished loading (whereas it would currently be displayed when at least *one* has finished loading)

Reply:
https://github.com/shapeshift/web/pull/5673/commits/9c1b6b8bef4408941e4e2bc2cda6683593d81c55